src/icons/User.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import colors from 'src/styles/colors'
+
+interface Props {
+  color?: string
```

Comment:
```suggestion
  color?: Colors
```
i also wonder if we want to use this User icon with the CircledIcon for DefaultAvatar, these look similar

Reply:
They're nearly imperceptibly different -- I'll replace the default avatar with this, since this is the newest version.